panel/widgets/slider.py

```diff
@@ -351,13 +353,50 @@ def values(self):
         return list(self.options.values()) if isinstance(self.options, dict) else self.options
 
 
-class RangeSlider(_SliderBase):
+
+class _RangeSliderBase(_SliderBase):
+
+    value = param.Tuple(length=2)
+
+    value_start = param.Parameter()
+
+    value_end = param.Parameter()
+
+    __abstract = True
+
+    def __init__(self, **params):
+        if 'value' not in params:
+            params['value'] = (params.get('start', self.start),
+                               params.get('end', self.end))
+        params['value_start'], params['value_end'] = params['value']
+        super().__init__(**params)
+
+    @param.depends('value', watch=True)
+    def _sync_values(self):
+        vs, ve = self.value
+        with edit_readonly(self):
+            self.param.set_param(value_start=vs, value_end=ve)
```

Comment:
Were value_start and value_end meant to be readonly, then?

Reply:
Hmm, yeah this is wrong. Thanks for catching it, if I can I will make it read-only again.

Reply:
I'm not sure it's feasible for them to be read only if you want them to be independently controlled by widgets. Seems like it would be good to see if param.Composite can be updated to work with events and dependencies, as it was meant for precisely this use case...